package.json

```diff
@@ -92,17 +91,21 @@
         "webpack-dev-server": "^4.5.0"
     },
     "peerDependencies": {
-        "react": "^16.9.0",
-        "react-dom": "^16.9.0"
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0"
```

Comment:
how does bumping the react version affect simularium-website?

Reply:
everything seems to work fine, but it will necessitate us bumping the react version on the front end too. I was able to do it pretty quickly on the front end with a local installation. The only other dependency that had to be changed was a "react-waypoints" dep.

Reply:
if you make this look too easy, you will remove any excuses we have for not updating other codebases